com.unity.netcode.gameobjects/Runtime/NetworkVariable/Collections/NetworkDictionary.cs

```diff
@@ -127,33 +109,6 @@ public override void ReadDelta(Stream stream, bool keepDirtyDelta)
                             }
                         }
                         break;
-                    case NetworkDictionaryEvent<TKey, TValue>.EventType.RemovePair:
```

Comment:
I can't see a good reason for RemovePair.  This was a holdover from implementing IDictionary.  We didn't actually do anything special like deal with if you remove an existing key with a mismatched value, and to me implementing this is unnecessarily complicated, hence out it goes.